cylc/sphinx_ext/cylc_lang/autodocumenters.py

```diff
@@ -346,3 +367,314 @@ def run(self):
+        for section, select_from in zip(
+            ['platforms', 'platform groups'],
+            ['hosts', 'platforms']
+        ):
+            metadata[section] = config.get(section)
+            if not metadata[section]:
+                continue
+            for key in config.get(section).keys():
+                # Grab a list of hosts or platforms that this
+                # platform or group will select from:
+                select_from = (
+                    config.get(section).get(key).get('hosts')
+                    or config.get(section).get(key).get('platforms'))
+                select_from = select_from or [key]
+                metadata[section][key]['select_from'] = select_from
+
+                # Grab the job runner this platform uses:
+                if section == 'platforms':
+                    metadata[section][key]['job_runner'] = config.get(
+                        section).get(key).get('job runner', 'background')
+        return metadata
+
+    @staticmethod
+    def global_to_node(meta, src):
+        """Convert the global metadata into rst format.
+        """
+        ret = [
+            f'.. cylc:conf:: Global Config: {src}', '',
+            f'{INDENT}.. note::', '',
+            f'{INDENT * 2}Platforms and platform groups are listed'
+            ' in the order in which Cylc will check for matches to the'
+            ' ``[runtime][NAMESPACE]platform`` setting.', '']
+
+        # For platforms in platform groupts then for hosts in platforms:
+        for settings, selects in zip(
+            ['platform groups', 'platforms'],
+            ['platforms', 'hosts']
+        ):
+            if meta.get(settings, {}):
+                ret += [f'{INDENT}.. cylc:conf:: {settings}', '']
+                for regex, info in reversed(meta[settings].items()):
+                    platform_meta = info.get('meta', {})
+                    title = platform_meta.get('title', '')
+                    if not title:
+                        title = regex
+                    ret += [
+                        f'{INDENT * 3}.. cylc:conf:: {title}', '',
+                        f'{INDENT * 4}.. csv-table:: Key Details', '']
+
+                    # Add the definition regex if an explicit title
+                    # has been given to the platform.
+                    if title != regex:
+                        ret.append(f'{INDENT * 5}Regex, ``{regex}``.')
+
+                    if info.get('job_runner'):
+                        ret.append(
+                            f'{INDENT * 5}job runner,'
+                            f' ``{info.get("job_runner")}``')
+
+                    # Lists platforms in group or hosts in platform.
+                    selectables = '- ' + '\n- '.join(
+                        f'``{i}``' for i in info["select_from"])
+                    ret += [f'{INDENT * 5}{selects},"{selectables}"', '']
+
+                    ret.append('')
+                    url = platform_meta.get('url', '')
```

Comment:
The URL config uses all-caps not lower: https://cylc.github.io/cylc-doc/stable/html/reference/config/workflow.html#flow.cylc[meta]URL